src/modules/cpu/kernel/jitter.hpp

```diff
@@ -0,0 +1,757 @@
+#include "rppdefs.h"
+#include "rpp_cpu_simd.hpp"
+#include "rpp_cpu_common.hpp"
+
+RppStatus jitter_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                   RpptDescPtr srcDescPtr,
+                                   Rpp8u *dstPtr,
+                                   RpptDescPtr dstDescPtr,
+                                   Rpp32u *kernelSizeTensor,
+                                   RpptXorwowStateBoxMuller *xorwowInitialStatePtr,
+                                   RpptROIPtr roiTensorPtrSrc,
+                                   RpptRoiType roiType,
+                                   RppLayoutParams layoutParams)
+{
+    RpptROI roiDefault = {0, 0, (Rpp32s)srcDescPtr->w, (Rpp32s)srcDescPtr->h};
+
+    omp_set_dynamic(0);
+#pragma omp parallel for num_threads(dstDescPtr->n)
```

Comment:
We need to use numThreads from handle instead of using dstDescPtr->n
Please refer below link and make the same changes
https://github.com/r-abishek/rpp/blob/f9a7063d19c06cc9e273353fd1d2b5719cedcaae/src/modules/cpu/kernel/brightness.hpp#L41

Reply:
Done

Reply:
Done